lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -35,7 +55,7 @@ export default function Dialog({
   buttons,
 }) {
   const dialogTitleId = useUniqueId('dialog');
-  const rootEl = useRef();
+  const rootEl = useRef(/** @type {HTMLDivElement|null} */ (null));
 
   useElementShouldClose(rootEl, true, () => {
     if (typeof onCancel === 'function') {
```

Comment:
not. could be be simplified to `if(onCancel)` now?

seems like it would either be the right thing or undefined.

Reply:
Yes, I think so. I can't see a reason to use the more verbose check.